lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -256,6 +256,7 @@ def populate_api_routes( webapp, app ):
     webapp.mapper.resource( 'remote_file', 'remote_files', path_prefix='/api' )
     webapp.mapper.resource( 'group', 'groups', path_prefix='/api' )
     webapp.mapper.resource_with_deleted( 'quota', 'quotas', path_prefix='/api' )
+    webapp.mapper.connect( '/api/interactive_environment_ready', controller='interactive_environment_ready' )
```

Comment:
I'd consider a web controller for this instead of an API controller - it is what I think we use for something like getting the current history ID that isn't stateless or RESTful.